### ISSUE TITLE
The Skip link does not receive focus when it is tabbed to

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#131]
Screengrab/videos (insert file names here)
2020 08 18, Skip link lacks focus.mp4
[2020 08 18, Skip link lacks focus.docx.zip](https://github.com/CityOfBoston/Iterators/files/5098899/2020.08.18.Skip.link.lacks.focus.docx.zip)

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 84

Trusted Tester Test
Focus - Visible

Brief Description
The Skip link does not receive focus when it is tabbed to

Steps to reproduce
1)	Tab to the coronavirus information button
2)	00:02 Tab to the next tabbable area (‘skip link’)
3)	00:07 Tab to before and after the skip link
a)	Note: both areas before and after the skip link show focus when tabbed to 

Desired Result
•	All buttons and links when tabbed to have visible focus

Actual Result
•	The skip link does not have visible focus when tabbed to

**Comment by ollie-iterators** [comment]
This is a duplicate of a previously reported bug
